packages/mobile/src/screens/profile-screen/TracksTab.tsx

```diff
@@ -10,6 +13,18 @@ import { getProfile } from './selectors'
 export const TracksTab = () => {
   const { profile } = useSelectorWeb(getProfile)
   const lineup = useSelectorWeb(getProfileTracksLineup)
+  const dispatchWeb = useDispatchWeb()
+
+  const loadMore = useCallback(
+    (offset: number, limit: number) => {
+      dispatchWeb(
+        tracksActions.fetchLineupMetadatas(offset, limit, false, {
```

Comment:
what happens if this is null?

Reply:
ah, we do get the existing issue, so will add a check here just in case :)